Sources/SUIComponents/Molecules/StatusView.swift

```diff
@@ -97,19 +97,22 @@ extension Components.Molecules.StatusView {
         public let body: String?
         public let buttonModel: ButtonModel?
         public let iconTintColor: Color?
+        public let defaultTextAligment: TextAlignment?
```

Comment:
Should this be named textAlignment or bodyTextAlignment ?

Reply:
yes I want it to convey default. the statusView should always be center. EXCEPT when the text is over 2 lines long according to design, so you're being specific that you want the default to be something other than center.